### ISSUE TITLE
For some reason, my device cannot be found

### ISSUE THREAD

**Issue by edasque** [otothea/node-blync-api#1]
On my Mac, this: `return dev.vendorId === 4400 && dev.productId === 1` filters out my device.

My productID seems to be 7680 (and for some reason it comes up twice in my HID list):
  { vendorId: 4400,
    productId: 7680,
    path: 'USB_1130_1e00_14100000',
    serialNumber: '',
    manufacturer: 'TENX',
    product: 'Blync',
    release: 1320,
    interface: -1 },

Any idea why that would be?

**Comment by otothea** [comment]
Nope, but we could move the product id to the config file or something

**Comment by edasque** [comment]
Well, I did just change the test but the app never changes the color. I wonder if the command set is slightly different for that device.

**Comment by otothea** [comment]
Yeah, interestingly enough, the Blync device I have is not product: 'Blync'. Mine is iBuddy: http://ibuddylib.sourceforge.net/ I wonder if they are manufacturing a different product now?

**Comment by edasque** [comment]
Ah, interesting. That could explain why mine won't light up at all. I am guessing the commands might be different. How do you think one could reverse engineer the commands on a Mac?

**Comment by otothea** [comment]
The first thing I would do is contact Blync and see if they have any documentation they could give you, or point you in the right direction.
